hub/graphql/types/model_types.py

```diff
@@ -880,7 +894,7 @@ def is_update_scheduled(self: models.ExternalDataSource, info: Info) -> bool:
     def update_progress(
         self: models.ExternalDataSource, info: Info
     ) -> Optional[BatchJobProgress]:
-        job = self.get_scheduled_update_job()
+        job = self.get_latest_update_job()
         if job is None:
             return None
         progress = self.get_scheduled_batch_job_progress(job)
```

Comment:
Do you need to pass the `user` in here, too?

Reply:
I think I've added that now